packages/js/src/feeds/feeds.ts

```diff
@@ -43,28 +48,30 @@ export class Feeds extends BaseModule {
     });
   }
 
-  async fetchCount(countArgs: FetchCountArgs = {}): Promise<FetchCountResponse> {
+  async fetchCount(args?: FetchFilterCountArgs): Promise<FetchFilterCountResponse>;
+  async fetchCount(args?: FetchFiltersCountArgs): Promise<FetchFiltersCountResponse>;
+  async fetchCount(args: FetchCountArgs): Promise<FetchCountResponse> {
```

Comment:
the supported calls are like:
```
// get all unread/read, archived/unarchived notifications counts
const countResponse = await novu.feeds.fetchCount(); 

// get unread notifications counts
const countResponse = await novu.feeds.fetchCount({ read: false }); 

// get unread counts and archived counts with tag newsletter
const countResponse = await novu.feeds.fetchCount({
   filters: [{ read: false }, { tags: ["newsletter"], archived: false }]
}); 

```

Reply:
The parameters work really nicely. Should we rename `fetchCount` to just `count` in a more ORM like fashion?